src/odemis/gui/cont/tabs.py

```diff
@@ -3254,9 +3260,25 @@ def _display_meteor_pos_warning_msg(self, end_pos) -> bool:
                     pos_str.append(f"{axis} = " + readable_str(math.degrees(end_pos[axis]), "°", 4))
                 else:
                     pos_str.append(f"{axis} = " + readable_str(end_pos[axis], "m", 4))
-        pos_str = "\n". join(pos_str)
+        pos_str = """\n""". join(pos_str)
+
+        # Check the deviation in rotation angle when switching from SEM to FM,
+        # give a warning message if switching is done from a different rotation angle
+        target_label = self.tab_data_model.main.posture_manager.getCurrentPostureLabel(end_pos)
+        warn_msg = STAGE_WARN_MSG
```

Comment:
@pieleric I have made a global variable for the the multiline message. If the same multiline message needs to be included in this function, it looks like below
![image](https://github.com/delmic/odemis/assets/37955981/576e7880-f76d-4bc1-b09e-0476e510c36e)